python/ray/includes/global_state_accessor.pxi

```diff
@@ -43,6 +43,12 @@ cdef class GlobalStateAccessor:
             result = self.inner.get().GetAllJobInfo()
         return result
 
+    def get_next_job_id(self):
+        cdef CJobID cjob_id
+        with nogil:
+            cjob_id = self.inner.get().GetNextJobID()
+        return cjob_id.ToInt()
```

Comment:
Why do we need `.ToInt()` here?

Reply:
The python level cannot understand CJobID.

Reply:
Then why not return a Python JobID here?

Reply:
The `inner` variable is a CGlobalStateAccessor pointer which **cimport** from `global_state_accessor.h`, so it cannot return Python JobID.

Reply:
Sorry, I don't think I understand. Can we do `JobID(cjob_id.Binary())` here?

Reply:
Yes maybe we can. But I think the better way is to keep the current change and do serialize part in `GlobalState` in state.py, like other interfaces.
BTW I'd rather use `.ToInt` as the origin value is from an integer type value.